consensus/src/dag/dag_driver.rs

```diff
@@ -113,7 +113,9 @@ impl DagDriver {
             driver.broadcast_node(node);
         } else {
             // kick start a new round
-            block_on(driver.enter_new_round(highest_strong_links_round + 1));
+            if !driver.dag.read().is_empty() {
```

Comment:
Good question...

Reply:
I probably need to rename the function lol, is_empty actually checks that condition inside

Reply:
condition is `self.start_round > 1` doesn't match == 1.

Reply:
hmm, classic bug lol. but if it's 1, is_empty should be true and we need to sync the dag not enter new round?

Reply:
I see now. when `start_round == 1`, you consider the DAG is not empty (which is very confusing). So, its the function name that needs to change.

Reply:
it's a bug, it should be start_round > 0, I'll update the pr when i have better wifi. when the dag is NOT empty we can enter new round otherwise we need to sync

Reply:
We are confused and do not see a bug now... 
(The dag starts from round 1)

Reply:
ah, I was confused too. I thought it started from 0, so the code is fine